src/autocomplete/databases/postgresql/postgresql-extract-statements.ts

```diff
@@ -63,10 +63,11 @@ export function extractPostgreSqlStatementsFromQuery(
                 startLine,
                 endLine: lastStatementToken.line,
                 startColumn,
-                endColumn: lastStatementToken.column + (lastStatementToken.text?.length || 0),
+                endColumn:
+                    lastStatementToken.columnIndex + 1 + getTokenTextLength(lastStatementToken),
```

Comment:
still why do we need to call this function twice? this is the same variable as far as I can tell
you can just put it like this

`const endColumn = token.text?.length || 0`